packages/components/src/TestComponent/TestComponent.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import { Story } from "@storybook/react/types-6-0";
+import TestComponent from "./TestComponent";
```

Comment:
NOTE: This import is not automatically alphabetized by the generator, so, in this case, I had to manually move the component import down one line to be below `Story`.

Reply:
dang that's kinda annoying. we do have a lint sort-imports rule setup but I'm not sure it can auto-fix 🤔   def can be a future followup

Reply:
Should I make a ticket?

Reply:
that'd be great, thanks! we can put it in the [DS: Strategy/Infra](https://app.clubhouse.io/czi-edu/epic/53445/design-system-strategy?vc_group_by=day) epic

Reply:
https://app.clubhouse.io/czi-edu/story/129076/make-generated-plop-story-file-auto-sort-imports

Reply:
There is an autofix version though https://www.npmjs.com/package/eslint-plugin-simple-import-sort.